install_config/registry/deploy_registry_existing_clusters.adoc

```diff
@@ -91,7 +94,147 @@ Daemonsets ensure that when nodes are created, they contain copies of a
 specified pod. When the nodes are removed, the pods are garbage collected.
 
 For more information on `DaemonSets`, see
-xref:../../dev_guide/daemonsets.adoc#dev-guide-daemonsets[Using Daemonsets].
+xref:../../dev_guide/daemonsets.adoc#dev-guide-daemonsets[Using DaemonSets].
+
+[[redeploy-registry]]
+=== Re-deploying the Registry
+
+There is a couple of things to keep on mind when re-deploying an existing docker-registry
+DeploymentConfig or DaemonSet:
+
+1. Service IP shall be preserved.
```

Comment:
unclear what this means.  Do you mean mean the service ip will not be changed as a result of redeploying, or is this a command to the user ("don't change the service ip when you redeploy")?

Reply:
The latter. It's very easy to change the service IP and it should be avoided. I'll try to be more clear.